pkg/server/commands/write_authzmodel.go

```diff
@@ -13,20 +13,28 @@ import (
 
 // WriteAuthorizationModelCommand performs updates of the store authorization model.
 type WriteAuthorizationModelCommand struct {
-	backend storage.TypeDefinitionWriteBackend
-	logger  logger.Logger
+	backend       storage.TypeDefinitionWriteBackend
+	logger        logger.Logger
+	allowSchema10 bool
 }
 
 func NewWriteAuthorizationModelCommand(
 	backend storage.TypeDefinitionWriteBackend,
 	logger logger.Logger,
+	allowSchema10 bool,
 ) *WriteAuthorizationModelCommand {
 	return &WriteAuthorizationModelCommand{
-		backend: backend,
-		logger:  logger,
+		backend:       backend,
+		logger:        logger,
+		allowSchema10: allowSchema10,
 	}
 }
 
+// IsAuthorizationModelObsolete returns whether the model schema is allowed or it is obsolete
+func IsAuthorizationModelObsolete(schemaVersion string, allowSchema10 bool) bool {
+	return !allowSchema10 && schemaVersion == typesystem.SchemaVersion1_0
+}
+
```

Comment:
this function is used by multiple commands so why is it defined in this command?

Reply:
moved to a separate file